charts/renku/templates/tests/test-renku.yaml

```diff
@@ -1,51 +1,42 @@
 apiVersion: v1
 kind: Pod
```

Comment:
Would we gain anything in terms of persisting error-state, etc from using a k8s job instead of a naked pod here? I actually don't think so but I'm not sure.

Reply:
I've tested and found out that while it is possible to use a job it seems we'd have nothing to gain from it, so I'd go for the simplest solution of the naked pod, which is also what's used in the Helm test documentation example.